LocalSettings.php

```diff
@@ -4672,31 +4676,9 @@
 				'editinterface' => true,
 			),
 		),
-		'+whentheycrywiki' => array(
-			'user' => array(
-				'createtalk',
-				'edit-create',
-			),
-		),
-		'+wikipucwiki' => array(
+		'+vsfan' => array(
 			'*' => array(
-				'ajaxpoll-vote' => true,
-				'ajaxpoll-view-results' => true,
-				'voteny' => true,
-				'upload' => true,
-			),
-		),
-		'+yeoksawiki' => array(
-			'sysop' => array(
-				'project-edit' => true,
-			),
-		),
-		'+zhdelwiki' => array(
-			'confirmed' => array(
-				'createpage' => true,
-				'edit' => true,
-				'move' => true,
-				'upload' => true,
+				'edit' => false,
```

Comment:
=> false doesn't work in this variable.

Also please use the relevant $wmgEditingMatrix config